mula/scheduler/connectors/services/bytes.py

```diff
@@ -63,7 +63,7 @@ def _get_token(self) -> str:
         response = self.post(
             url=url,
             payload=self.credentials,
-            headers={"Content-Type": "application/x-www-form-urlendcoded"},
+            headers={"Content-Type": "application/x-www-form-urlencoded"},
```

Comment:
When you use a dict-like object for the `data` argument in `requests`, it will automatically set this header.

```suggestion
```